gatling/gatling.py

```diff
@@ -193,27 +201,34 @@ def sink(selector, not_done):
                 if p._n >= max_bytes:
                     t1 = time()
                     p._t1 = t1
-                    td = (t1-t0) * 1024
+                    td = (t1 - t0) * 1024
                     ptd = (t1 - p._t0) * 1024
                     p.stdin.close()
                     p_filled.append(p)
                     if verbose and ptd and td:
                         running = len(p_filled) + len(p_open)
-                        print(f"# {my_name} PROCESS LIMIT {p._n:,}/{max_bytes:,}",
-                              f"({p._n/ptd:.2f} kb/s).",
-                              f"INPUT: {b_in:,} ({b_in/td:.2f} kb/s) OUTPUT: {not_done[0]:,}.",
-                              f"PROCESSES: {running}/{running+len(res)}",
-                              file=sys.stderr)
+                        print(
+                            f"# {my_name} PROCESS LIMIT {p._n:,}/{max_bytes:,}",
+                            f"({p._n/ptd:.2f} kb/s).",
+                            f"INPUT: {b_in:,} ({b_in/td:.2f} kb/s) OUTPUT: {not_done[0]:,}.",
+                            f"PROCESSES: {running}/{running+len(res)}",
+                            file=sys.stderr,
+                        )
                     while len(p_filled) == max_procs:
                         done = [d for d in p_filled if d.poll() is not None]
                         if done and verbose:
-                            print(f"# {my_name} CLOSED {len(done)} PROCESSES", file=sys.stderr)
+                            print(
+                                f"# {my_name} CLOSED {len(done)} PROCESSES",
+                                file=sys.stderr,
+                            )
```

Comment:
Can someone honestly debate that this is not a step back in readability?

Reply:
This is actually the approach that I take whenever parameters cause me to exceed the line length limit. Apparently it takes a little getting used to but it does grow on you (at least, according to other folks who have decided to adopt my strange ways ;))

Reply:
What line length is yapf set to? You can actually set a line length with black (its one of the few options it has)

Reply:
Default, so 88